apps/cli/src/commands/shell.ts

```diff
@@ -41,12 +42,17 @@ export default class Shell extends Command {
             sdkImage,
             "cartesi-machine",
             `--ram-length=${ramSize}`,
-            "--rollup",
             `--flash-drive=label:${driveLabel},filename:${ext2}`,
-            "-i",
         ];
+
         if (runAsRoot) {
-            args.push("--append-rom-bootargs='single=yes'");
+            args.push("--append-init=USER=root");
+        }
+
+        if (!(await lookpath("stty"))) {
```

Comment:
what is this for?

Reply:
I use this to check if `stty` is available, and if it is I enable the `-it` option.

> -it option for improved terminal experience was added (colored, support icons, and bigger size), ideally Sunodo should use it, however this depends on stty command being available in the host.